internal/cmd/destinations.go

```diff
@@ -47,7 +47,7 @@ func newDestinationsListCmd(cli *CLI) *cobra.Command {
 		Short:   "List connected destinations",
 		Args:    NoArgs,
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			client, err := defaultAPIClient()
+			client, err := defaultAPIClient(cmd.Flags())
```

Comment:
Instead of passing `cmd.Flags()` would it make sense to just have `cli` include this state? E.g.

```
client, err := defaultAPIClient(cli.IgnoreVersion)
```

Ideally it's optional somehow (perhaps by making `defaultAPIClient` take an options struct). And then `rootPreRun` in `cmd.go` could parse the value and include it in the `CLI` struct that's passed around

Reply:
With above comment, we could have something like:

```
client := api.Client{IgnoreVersion: cli.IgnoreVersion}
```